sdk/src/transaction/sanitized.rs

```diff
@@ -67,6 +72,7 @@ impl SanitizedTransaction {
         address_loader: impl AddressLoader,
     ) -> Result<Self> {
         let signatures = tx.signatures;
+        let transaction_meta = tx.transaction_meta;
```

Comment:
`transaction_meta` is moved over if `SanitizedTransaction` is created from `SanitizedVersionedTransaction`